src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/SimpleTypeInfos.cs

```diff
@@ -292,9 +292,12 @@ public override void WriteData(PropertyValue value)
         {
             // It's not currently possible to get the HasValue property of a nullable type through reflection when the
             // value is null. Instead, we simply check that the nullable is not null.
-            bool hasValue = value.ReferenceValue != null;
+            object? refVal = value.ReferenceValue;
```

Comment:
Can you also remove the corresponding suppressions from the xml files?

https://github.com/dotnet/runtime/blob/5d5c3e7a589e21e43ad6464b6ed6732e98e7fc8a/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Suppressions.Shared.xml#L76-L81

https://github.com/dotnet/runtime/blob/5d5c3e7a589e21e43ad6464b6ed6732e98e7fc8a/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Suppressions.Shared.xml#L94-L99